app/views/courses/_status_panel.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="status-box">
+  <h3 class="govuk-heading-m">Status</h3>
+  <div class="govuk-!-margin-bottom-6">
+    <%= render partial: 'content_status_tag', locals: { course: @course } %>
+  </div>
+  <h3 class="govuk-heading-s govuk-!-margin-bottom-0">Is it on Find?</h3>
+  <% if @course.findable? %>
+    <p class="govuk-body">
+      <%= govuk_link_to "Yes - view online", search_ui_course_page_url(provider_code: @provider.provider_code, course_code: @course.course_code) %>
+    </p>
+  <% else %>
+    <p class="govuk-body">
+      <% if @course.new_and_not_running? %>
+        No – <%= @provider.opted_in? ? 'still in draft' : 'must be published on UCAS' %>
+      <% else %>
+        No
+      <% end %>
```

Comment:
It's logic like this in the view that makes me want to just get the `draper` gem in here. :(